### ISSUE TITLE
Web client does not react to Parental Access Schedule

### ISSUE THREAD

**Issue by JPVenson** [jellyfin/jellyfin-web#4447]
**Describe The Bug**
The web client does not react properly to the expiration of the configured Access Schedule mid session. While logged in, when the configured timeframe expires, the application does only show endless waiter animations. This is also true if the user has a persisted login and reopens the webpage when outside the configured access schedule.

**Steps To Reproduce**
1. Login to user 'A'
2. Close Browser
3. Configure access schedule for user 'A' to be outside current time. (e.g. Current time is 1PM and user A is only allowed to access during 6AM to 8AM)
4. reopen client with persisted user A's login
5. Endless loading animation

Alt:
1. Configure access schedule for user 'A' to be allowed access from 1PM to 2PM
2. Login to user 'A' at 1:59PM
3. wait 2 minutes until 2:01PM while leaving browser open
4. Try to open any page with User A
5. Endless loading animation

**Expected Behavior**
Either an automatic logoff or a notification that tells the user he has no longer access

**Logs**

**Screenshots**

**System (please complete the following information):**
 - Platform: [e.g. Linux, Windows, iPhone, Tizen]
 - Browser: Edge, JMP
 - Jellyfin Version: 10.8.9

**Comment by Shadowghost** [comment]
I think this is likely caused by the general problem of the webclient to not properly handle 401 responses gracefully.

**Comment by JPVenson** [comment]
I agree, this also includes other unexpected errors like 501s where the clients just hangs itself
